src/_data/authors.json

```diff
@@ -174,5 +174,15 @@
         "name": "Bartosz Wojtkiewicz",
         "bio": "Bartek is a Principal Software Engineer at Allegro who is working on making Allegro Platform ready for international customers. He tries to make a living as a T-Shaped engineer with a solid background in backend engineering. Enjoys pandemic lockdowns and obscure japanese games on Nintendo Switch.",
         "linkedin": "bartosz-wojtkiewicz-11585658"
+    },
+    "artur.chabera": {
+        "name": "Artur Chabera",
+        "bio": "Artur has been with Allegro for 2 years. His main focus is the area of payments and the iOS purchase process. He likes to look at the application through the prism of accessibility. He likes to spend his free time riding a bike and reading sci-fi novels.",
```

Comment:
through the prism of accessibility -> with a focus on accessibility

Reply:
I think using the expression with "prism" is correct here. A similar example is mentioned here https://dictionary.cambridge.org/dictionary/english/prism